Skip to:
  * [Skip to content](https://www.igbb.msstate.edu/citations.php#page)
  * [Skip to navigation](https://www.igbb.msstate.edu/citations.php#main-menu)


Banner
[](https://www.igbb.msstate.edu/index.php)
# Highly Cited Papers
Over the years, IGBB scientists have gained a reputation for publishing papers that are widely cited. Information regarding the 100 most highly cited papers authored/co-authored by IGBB employees, faculty fellows, and affiliates is presented below. Citation values are from [Scopus](https://www.scopus.com/). For a particular IGBB fellow/affiliate/staff member, only those papers published while at MS State are included.
  
Page 1 of 9 [](https://www.igbb.msstate.edu/citations.php?page=2) [](https://www.igbb.msstate.edu/citations.php?page=9)  
  

The Sorghum bicolor genome and the diversification of grasses  
  

IGBB Authors:  
Daniel G. Peterson
PUBLICATION YEAR: 2009  
IMPACT FACTOR: 17.621  
CITATION COUNT: **[2498](http://www.scopus.com/inward/citedby.url?partnerID=40&md5=8ab0f0680cf3197f71796d320074d996&origin=inward&eid=2-s2.0-58449137410)**  

more
  
  
  

  

Paterson AH, Bowers JE, Bruggmann R, Dubchak I, Grimwood J, Gundlach H, Haberer G, Hellsten U, Mitros T, Poliakov A, Schmutz J, Spannagl M, Tang H, Wang X, Wicker TM, Bharti AK, Chapman J, Feltus FA, Gowik U, Grigoriev IV, Lyons E, Maher CA, Martis M, Narechania A, Otillar RP, Penning BW, Salamov AA, Wang Y, Zhang L, Carpita NC, Freeling M, Gingle AR, Hash CT, Keller B, Klein P, Kresovich S, McCann MC, Ming R, Peterson DG, Rahman MU, Ware D, Westhoff P, Mayer KF, Messing J, Rokhsar DS (2009) The Sorghum bicolor genome and the diversification of grasses. Nature  
  
DOI[10.1038/nature07723](https://doi.org/10.1038/nature07723)  
EID[2-s2.0-58449137410](https://www.scopus.com/inward/record.uri?eid=2-s2.0-58449137410&doi=10.1038/nature07723&origin=inward)  
PMID[19189423](http://www.ncbi.nlm.nih.gov/pubmed/19189423)  
  
[DOWNLOAD PDF](https://www.igbb.msstate.edu/downloads/Paterson_et_al_2009.pdf)  
  
ABSTRACT  
Sorghum, an African grass related to sugar cane and maize, is grown for food, feed, fibre and fuel. We present an initial analysis of the approximately 730-megabase Sorghum bicolor (L.) Moench genome, placing approximately 98% of genes in their chromosomal context using whole-genome shotgun sequence validated by genetic, physical and syntenic information. Genetic recombination is largely confined to about one-third of the sorghum genome with gene order and density similar to those of rice. Retrotransposon accumulation in recombinationally recalcitrant heterochromatin explains the approximately 75% larger genome size of sorghum compared with rice. Although gene and repetitive DNA distributions have been preserved since palaeopolyploidization approximately 70 million years ago, most duplicated gene sets lost one member before the sorghum-rice divergence. Concerted evolution makes one duplicated chromosomal segment appear to be only a few million years old. About 24% of genes are grass-specific and 7% are sorghum-specific. Recent gene and microRNA duplications may contribute to sorghum's drought tolerance.  

  
  

Gene Ontology Consortium: going forward  
  

IGBB Authors:  
Shane C. Burgess, Teresia J. Buza, Catherine R. Gresham, Fiona M. McCarthy, Lakshmi Pillai
PUBLICATION YEAR: 2015  
IMPACT FACTOR: 10.031  
CITATION COUNT: **[2700](http://www.scopus.com/inward/citedby.url?partnerID=40&md5=8ab0f0680cf3197f71796d320074d996&origin=inward&eid=2-s2.0-84946735654)**  

more
  
  
  

  

Blake JA, Christie KR, Dolan ME, Drabkin HJ, Hill DP, Ni L, Sitnikov D, Burgess S, Buza T, Gresham C, McCarthy F, Pillai L, Wang H, Carbon S, Dietze H, Lewis SS, Mungall CJ, Munoz-Torres MC, Feuermann M, Gaudet P, Basu S, Chisholm RL, Dodson RJ, Fey P, Mi H, Thomas PD, Muruganujan A, Poudel S, Hu J, Aleksander SA, McIntosh BK, Renfro DP, Siegele DA, Attrill H, Brown NH, Tweedie S, Lomax J, Osumi-Sutherland D, Parkinson H, Roncaglia P, Lovering RC, Talmud PJ, Humphries SE, Denny P, Campbell NH, Foulger RE, Chibucos MC, Gwinn Giglio M, Chang HY, Finn R, Fraser M, Mitchell A, Nuka G, Pesseat S, Sangrador A, Scheremetjew M, Young SY, Stephan R, Harris MA, Oliver SG, Rutherford K, Wood V, Bahler J, Lock A, Kersey PJ, McDowall MD, Staines DM, Dwinell M, Shimoyama M, Laulederkind S, Hayman GT, Wang SJ, Petri V, D'Eustachio P, Matthews L, Balakrishnan R, Binkley G, Cherry JM, Costanzo MC, Demeter J, Dwight SS, Engel SR, Hitz BC, Inglis DO, Lloyd P, Miyasato SR, Paskov K, Roe G, Simison M, Nash RS, Skrzypek MS, Weng S, Wong ED, Berardini TZ, Li D, Huala E, Argasinska J, Arighi C, Auchincloss A, Axelsen K, Argoud-Puy G, Bateman A, Bely B, Blatter MC, Bonilla C, Bougueleret L, Boutet E, Breuza L, Bridge A, Britto R, Casals C, Cibrian-Uhalte E, Coudert E, Cusin I, Duek-Roggli P, Estereicher A, Famiglietti L, Gane P, Garmiri P, Gos A, Gruaz-Gumowski N, Hatton-Ellis E, Hinz U, Hulo C, Huntley R, Jungo F, Keller G, Laiho K, Lemercier P, Lieberherr D, MacDougall A, Magrane M, Martin M, Masson P, Mutowo P, O'Donovan C, Pedruzzi I, Pichler K, Poggioli D, Poux S, Rivoire C, Roechert B, Sawford T, Schneider M, Shypitsyna A, Stutz A, Sundaram S, Tognolli M, Wu C, Xenarios I, Chan J, Kishore R, Sternberg PW, Van Auken K, Muller HM, Done J, Li Y, Howe D, Westerfield M (2015) Gene Ontology Consortium: going forward. Nucleic Acids Research  
  
DOI[10.1093/nar/gku1179](https://doi.org/10.1093/nar/gku1179)  
EID[2-s2.0-84946735654](https://www.scopus.com/inward/record.uri?eid=2-s2.0-84946735654&doi=10.1093/nar/gku1179&origin=inward)  
PMID[25428369](http://www.ncbi.nlm.nih.gov/pubmed/25428369)  
  
[DOWNLOAD PDF](https://www.igbb.msstate.edu/downloads/Blake_et_al_2015.pdf)  
  
ABSTRACT  
The Gene Ontology (GO; http://www.geneontology.org) is a community-based bioinformatics resource that supplies information about gene product function using ontologies to represent biological knowledge. Here we describe improvements and expansions to several branches of the ontology, as well as updates that have allowed us to more efficiently disseminate the GO and capture feedback from the research community. The Gene Ontology Consortium (GOC) has expanded areas of the ontology such as cilia-related terms, cell-cycle terms and multicellular organism processes. We have also implemented new tools for generating ontology terms based on a set of logical rules making use of templates, and we have made efforts to increase our use of logical definitions. The GOC has a new and improved web site summarizing new developments and documentation, serving as a portal to GO data. Users can perform GO enrichment analysis, and search the GO for terms, annotations to gene products, and associated metadata across multiple species using the all-new AmiGO 2 browser. We encourage and welcome the input of the research community in all biological areas in our continued effort to improve the Gene Ontology.  

  
  

The genome sequence of taurine cattle: a window to ruminant biology and evolution  
  

IGBB Authors:  
Erdogan Memili, Nelida Rodriguez-Osorio
PUBLICATION YEAR: 2009  
IMPACT FACTOR: 15.467  
CITATION COUNT: **[1000](http://www.scopus.com/inward/citedby.url?partnerID=40&md5=8ab0f0680cf3197f71796d320074d996&origin=inward&eid=2-s2.0-77951621301)**  

more
  
  
  

  

Elsik CG, Tellam RL, Worley KC, Gibbs RA, Muzny DM, Weinstock GM, Adelson DL, Eichler EE, Elnitski L, Guigo R, Hamernik DL, Kappes SM, Lewin HA, Lynn DJ, Nicholas FW, Reymond A, Rijnkels M, Skow LC, Zdobnov EM, Schook L, Womack J, Alioto T, Antonarakis SE, Astashyn A, Chapple CE, Chen HC, Chrast J, Camara F, Ermolaeva O, Henrichsen CN, Hlavina W, Kapustin Y, Kiryutin B, Kitts P, Kokocinski F, Landrum M, Maglott D, Pruitt K, Sapojnikov V, Searle SM, Solovyev V, Souvorov A, Ucla C, Wyss C, Anzola JM, Gerlach D, Elhaik E, Graur D, Reese JT, Edgar RC, McEwan JC, Payne GM, Raison JM, Junier T, Kriventseva EV, Eyras E, Plass M, Donthu R, Larkin DM, Reecy J, Yang MQ, Chen L, Cheng Z, Chitko-McKown CG, Liu GE, Matukumalli LK, Song J, Zhu B, Bradley DG, Brinkman FS, Lau LP, Whiteside MD, Walker A, Wheeler TT, Casey T, German JB, Lemay DG, Maqbool NJ, Molenaar AJ, Seo S, Stothard P, Baldwin CL, Baxter R, Brinkmeyer-Langford CL, Brown WC, Childers CP, Connelley T, Ellis SA, Fritz K, Glass EJ, Herzig CT, Iivanainen A, Lahmers KK, Bennett AK, Dickens CM, Gilbert JG, Hagen DE, Salih H, Aerts J, Caetano AR, Dalrymple B, Garcia JF, Gill CA, Hiendleder SG, Memili E, Spurlock D, Williams JL, Alexander L, Brownstein MJ, Guan L, Holt RA, Jones SJ, Marra MA, Moore R, Moore SS, Roberts A, Taniguchi M, Waterman RC, Chacko J, Chandrabose MM, Cree A, Dao MD, Dinh HH, Gabisi RA, Hines S, Hume J, Jhangiani SN, Joshi V, Kovar CL, Lewis LR, Liu YS, Lopez J, Morgan MB, Nguyen NB, Okwuonu GO, Ruiz SJ, Santibanez J, Wright RA, Buhay C, Ding Y, Dugan-Rocha S, Herdandez J, Holder M, Sabo A, Egan A, Goodell J, Wilczek-Boney K, Fowler GR, Hitchens ME, Lozado RJ, Moen C, Steffen D, Warren JT, Zhang J, Chiu R, Schein JE, Durbin KJ, Havlak P, Jiang H, Liu Y, Qin X, Ren Y, Shen Y, Song H, Bell SN, Davis C, Johnson AJ, Lee S, Nazareth LV, Patel BM, Pu LL, Vattathil S, Williams RL, Jr., Curry S, Hamilton C, Sodergren E, Wheeler DA, Barris W, Bennett GL, Eggen A, Green RD, Harhay GP, Hobbs M, Jann O, Keele JW, Kent MP, Lien S, McKay SD, McWilliam S, Ratnakumar A, Schnabel RD, Smith T, Snelling WM, Sonstegard TS, Stone RT, Sugimoto Y, Takasuga A, Taylor JF, Van Tassell CP, Macneil MD, Abatepaulo AR, Abbey CA, Ahola V, Almeida IG, Amadio AF, Anatriello E, Bahadue SM, Biase FH, Boldt CR, Carroll JA, Carvalho WA, Cervelatti EP, Chacko E, Chapin JE, Cheng Y, Choi J, Colley AJ, de Campos TA, De Donato M, Santos IK, de Oliveira CJ, Deobald H, Devinoy E, Donohue KE, Dovc P, Eberlein A, Fitzsimmons CJ, Franzin AM, Garcia GR, Genini S, Gladney CJ, Grant JR, Greaser ML, Green JA, Hadsell DL, Hakimov HA, Halgren R, Harrow JL, Hart EA, Hastings N, Hernandez M, Hu ZL, Ingham A, Iso-Touru T, Jamis C, Jensen K, Kapetis D, Kerr T, Khalil SS, Khatib H, Kolbehdari D, Kumar CG, Kumar D, Leach R, Lee JC, Li C, Logan KM, Malinverni R, Marques E, Martin WF, Martins NF, Maruyama SR, Mazza R, McLean KL, Medrano JF, Moreno BT, More DD, Muntean CT, Nandakumar HP, Nogueira MF, Olsaker I, Pant SD, Panzitta F, Pastor RC, Poli MA, Poslusny N, Rachagani S, Ranganathan S, Razpet A, Riggs PK, Rincon G, Rodriguez-Osorio N, Rodriguez-Zas SL, Romero NE, Rosenwald A, Sando L, Schmutz SM, Shen L, Sherman L, Southey BR, Lutzow YS, Sweedler JV, Tammen I, Telugu BP, Urbanski JM, Utsunomiya YT, Verschoor CP, Waardenberg AJ, Wang Z, Ward R, Weikard R, Welsh TH, Jr., White SN, Wilming LG, Wunderlich KR, Yang J, Zhao FQ (2009) The genome sequence of taurine cattle: a window to ruminant biology and evolution. Science  
  
DOI[10.1126/science.1169588](https://doi.org/10.1126/science.1169588)  
EID[2-s2.0-77951621301](https://www.scopus.com/inward/record.uri?eid=2-s2.0-77951621301&doi=10.1126/science.1169588&origin=inward)  
PMID[19390049](http://www.ncbi.nlm.nih.gov/pubmed/19390049)  
  
[DOWNLOAD PDF](https://www.igbb.msstate.edu/downloads/Elsik_et_al_2009.pdf)  
  
ABSTRACT  
To understand the biology and evolution of ruminants, the cattle genome was sequenced to about sevenfold coverage. The cattle genome contains a minimum of 22,000 genes, with a core set of 14,345 orthologs shared among seven mammalian species of which 1217 are absent or undetected in noneutherian (marsupial or monotreme) genomes. Cattle-specific evolutionary breakpoint regions in chromosomes have a higher density of segmental duplications, enrichment of repetitive elements, and species-specific variations in genes associated with lactation and immune responsiveness. Genes involved in metabolism are generally highly conserved, although five metabolic genes are deleted or extensively diverged from their human orthologs. The cattle genome sequence thus provides a resource for understanding mammalian evolution and accelerating livestock genetic improvement for milk and meat production.  

  
  

Whole-genome analyses resolve early branches in the tree of life of modern birds  
  

IGBB Authors:  
David A. Ray
PUBLICATION YEAR: 2014  
IMPACT FACTOR: 18.914  
CITATION COUNT: **[1437](http://www.scopus.com/inward/citedby.url?partnerID=40&md5=8ab0f0680cf3197f71796d320074d996&origin=inward&eid=2-s2.0-84917708482)**  

more
  
  
  

  

Jarvis ED, Mirarab S, Aberer AJ, Li B, Houde P, Li C, Ho SY, Faircloth BC, Nabholz B, Howard JT, Suh A, Weber CC, da Fonseca RR, Li J, Zhang F, Li H, Zhou L, Narula N, Liu L, Ganapathy G, Boussau B, Bayzid MS, Zavidovych V, Subramanian S, Gabaldon T, Capella-Gutierrez S, Huerta-Cepas J, Rekepalli B, Munch K, Schierup M, Lindow B, Warren WC, Ray D, Green RE, Bruford MW, Zhan X, Dixon A, Li S, Li N, Huang Y, Derryberry EP, Bertelsen MF, Sheldon FH, Brumfield RT, Mello CV, Lovell PV, Wirthlin M, Schneider MP, Prosdocimi F, Samaniego JA, Vargas Velazquez AM, Alfaro-Nunez A, Campos PF, Petersen B, Sicheritz-Ponten T, Pas A, Bailey T, Scofield P, Bunce M, Lambert DM, Zhou Q, Perelman P, Driskell AC, Shapiro B, Xiong Z, Zeng Y, Liu S, Li Z, Liu B, Wu K, Xiao J, Yinqi X, Zheng Q, Zhang Y, Yang H, Wang J, Smeds L, Rheindt FE, Braun M, Fjeldsa J, Orlando L, Barker FK, Jonsson KA, Johnson W, Koepfli KP, O'Brien S, Haussler D, Ryder OA, Rahbek C, Willerslev E, Graves GR, Glenn TC, McCormack J, Burt D, Ellegren H, Alstrom P, Edwards SV, Stamatakis A, Mindell DP, Cracraft J, Braun EL, Warnow T, Jun W, Gilbert MT, Zhang G (2014) Whole-genome analyses resolve early branches in the tree of life of modern birds. Science  
  
DOI[10.1126/science.1253451](https://doi.org/10.1126/science.1253451)  
EID[2-s2.0-84917708482](https://www.scopus.com/inward/record.uri?eid=2-s2.0-84917708482&doi=10.1126/science.1253451&origin=inward)  
PMID[25504713](http://www.ncbi.nlm.nih.gov/pubmed/25504713)  
  
[DOWNLOAD PDF](https://www.igbb.msstate.edu/downloads/Jarvis_et_al_2014.pdf)  
  
ABSTRACT  
To better determine the history of modern birds, we performed a genome-scale phylogenetic analysis of 48 species representing all orders of Neoaves using phylogenomic methods created to handle genome-scale data. We recovered a highly resolved tree that confirms previously controversial sister or close relationships. We identified the first divergence in Neoaves, two groups we named Passerea and Columbea, representing independent lineages of diverse and convergently evolved land and water bird species. Among Passerea, we infer the common ancestor of core landbirds to have been an apex predator and confirm independent gains of vocal learning. Among Columbea, we identify pigeons and flamingoes as belonging to sister clades. Even with whole genomes, some of the earliest branches in Neoaves proved challenging to resolve, which was best explained by massive protein-coding sequence convergence and high levels of incomplete lineage sorting that occurred during a rapid radiation after the Cretaceous-Paleogene mass extinction event about 66 million years ago.  

  
  

Sequencing of allotetraploid cotton (Gossypium hirsutum L. acc. TM-1) provides a resource for fiber improvement  
  

IGBB Authors:  
Daniel G. Peterson
PUBLICATION YEAR: 2015  
IMPACT FACTOR: 25.478  
CITATION COUNT: **[1426](http://www.scopus.com/inward/citedby.url?partnerID=40&md5=8ab0f0680cf3197f71796d320074d996&origin=inward&eid=2-s2.0-84929171720)**  

more
  
  
  

  

Zhang T, Hu Y, Jiang W, Fang L, Guan X, Chen J, Zhang J, Saski CA, Scheffler BE, Stelly DM, Hulse-Kemp AM, Wan Q, Liu B, Liu C, Wang S, Pan M, Wang Y, Wang D, Ye W, Chang L, Zhang W, Song Q, Kirkbride RC, Chen X, Dennis E, Llewellyn DJ, Peterson DG, Thaxton P, Jones DC, Wang Q, Xu X, Zhang H, Wu H, Zhou L, Mei G, Chen S, Tian Y, Xiang D, Li X, Ding J, Zuo Q, Tao L, Liu Y, Li J, Lin Y, Hui Y, Cao Z, Cai C, Zhu X, Jiang Z, Zhou B, Guo W, Li R, Chen ZJ (2015) Sequencing of allotetraploid cotton (Gossypium hirsutum L. acc. TM-1) provides a resource for fiber improvement. Nature Biotechnology  
  
DOI[10.1038/nbt.3207](https://doi.org/10.1038/nbt.3207)  
EID[2-s2.0-84929171720](https://www.scopus.com/inward/record.uri?eid=2-s2.0-84929171720&doi=10.1038/nbt.3207&origin=inward)  
PMID[25893781](http://www.ncbi.nlm.nih.gov/pubmed/25893781)  
  
[DOWNLOAD PDF](https://www.igbb.msstate.edu/downloads/Zhang_et_al_2015)  
  
ABSTRACT  
Upland cotton is a model for polyploid crop domestication and transgenic improvement. Here we sequenced the allotetraploid Gossypium hirsutum L. acc. TM-1 genome by integrating whole-genome shotgun reads, bacterial artificial chromosome (BAC)-end sequences and genotype-by-sequencing genetic maps. We assembled and annotated 32,032 A-subgenome genes and 34,402 D-subgenome genes. Structural rearrangements, gene loss, disrupted genes and sequence divergence were more common in the A subgenome than in the D subgenome, suggesting asymmetric evolution. However, no genome-wide expression dominance was found between the subgenomes. Genomic signatures of selection and domestication are associated with positively selected genes (PSGs) for fiber improvement in the A subgenome and for stress tolerance in the D subgenome. This draft genome sequence provides a resource for engineering superior cotton lines.  

  
  

Repeated polyploidization of Gossypium genomes and the evolution of spinnable cotton fibres  
  

IGBB Authors:  
Daniel G. Peterson, Kurt C. Showmaker, William S. Sanders
PUBLICATION YEAR: 2012  
IMPACT FACTOR: 23.866  
CITATION COUNT: **[1056](http://www.scopus.com/inward/citedby.url?partnerID=40&md5=8ab0f0680cf3197f71796d320074d996&origin=inward&eid=2-s2.0-84871428041)**  

more
  
  
  

  

Paterson AH, Wendel JF, Gundlach H, Guo H, Jenkins J, Jin D, Llewellyn D, Showmaker KC, Shu S, Udall J, Yoo MJ, Byers R, Chen W, Doron-Faigenboim A, Duke MV, Gong L, Grimwood J, Grover C, Grupp K, Hu G, Lee TH, Li J, Lin L, Liu T, Marler BS, Page JT, Roberts AW, Romanel E, Sanders WS, Szadkowski E, Tan X, Tang H, Xu C, Wang J, Wang Z, Zhang D, Zhang L, Ashrafi H, Bedon F, Bowers JE, Brubaker CL, Chee PW, Das S, Gingle AR, Haigler CH, Harker D, Hoffmann LV, Hovav R, Jones DC, Lemke C, Mansoor S, Rahman M, Rainville LN, Rambani A, Reddy UK, Rong JK, Saranga Y, Scheffler BE, Scheffler JA, Stelly DM, Triplett BA, Van Deynze A, Vaslin MF, Waghmare VN, Walford SA, Wright RJ, Zaki EA, Zhang T, Dennis ES, Mayer KF, Peterson DG, Rokhsar DS, Wang X, Schmutz J (2012) Repeated polyploidization of Gossypium genomes and the evolution of spinnable cotton fibres. Nature  
  
DOI[10.1038/nature11798](https://doi.org/10.1038/nature11798)  
EID[2-s2.0-84871428041](https://www.scopus.com/inward/record.uri?eid=2-s2.0-84871428041&doi=10.1038/nature11798&origin=inward)  
PMID[23257886](http://www.ncbi.nlm.nih.gov/pubmed/23257886)  
  
[DOWNLOAD PDF](https://www.igbb.msstate.edu/downloads/Paterson_et_al_2012.pdf)  
  
ABSTRACT  
Polyploidy often confers emergent properties, such as the higher fibre productivity and quality of tetraploid cottons than diploid cottons bred for the same environments. Here we show that an abrupt five- to sixfold ploidy increase approximately 60 million years (Myr) ago, and allopolyploidy reuniting divergent Gossypium genomes approximately 1-2 Myr ago, conferred about 30-36-fold duplication of ancestral angiosperm (flowering plant) genes in elite cottons (Gossypium hirsutum and Gossypium barbadense), genetic complexity equalled only by Brassica among sequenced angiosperms. Nascent fibre evolution, before allopolyploidy, is elucidated by comparison of spinnable-fibred Gossypium herbaceum A and non-spinnable Gossypium longicalyx F genomes to one another and the outgroup D genome of non-spinnable Gossypium raimondii. The sequence of a G. hirsutum A(t)D(t) (in which 't' indicates tetraploid) cultivar reveals many non-reciprocal DNA exchanges between subgenomes that may have contributed to phenotypic innovation and/or other emergent properties such as ecological adaptation by polyploids. Most DNA-level novelty in G. hirsutum recombines alleles from the D-genome progenitor native to its New World habitat and the Old World A-genome progenitor in which spinnable fibre evolved. Coordinated expression changes in proximal groups of functionally distinct genes, including a nuclear mitochondrial DNA block, may account for clusters of cotton-fibre quantitative trait loci affecting diverse traits. Opportunities abound for dissecting emergent properties of other polyploids, particularly angiosperms, by comparison to diploid progenitors and outgroups.  

  
  

Butterfly genome reveals promiscuous exchange of mimicry adaptations among species  
  

IGBB Authors:  
Brian A. Counterman, David A. Ray
PUBLICATION YEAR: 2012  
IMPACT FACTOR: 23.866  
CITATION COUNT: **[881](http://www.scopus.com/inward/citedby.url?partnerID=40&md5=8ab0f0680cf3197f71796d320074d996&origin=inward&eid=2-s2.0-84863496231)**  

more
  
  
  

  

Dasmahapatra KK, Walters JR, Briscoe AD, Davey JW, Whibley A, Nadeau NJ, Zimin AV, Hughes DST, Ferguson LC, Martin SH, Salazar C, Lewis JJ, Adler S, Ahn SJ, Baker DA, Baxter SW, Chamberlain NL, Chauhan R, Counterman BA, Dalmay T, Gilbert LE, Gordon K, Heckel DG, Hines HM, Hoff KJ, Holland PWH, Jacquin-Joly E, Jiggins FM, Jones RT, Kapan DD, Kersey P, Lamas G, Lawson D, Mapleson D, Maroja LS, Martin A, Moxon S, Palmer WJ, Papa R, Papanicolaou A, Pauchet Y, Ray DA, Rosser N, Salzberg SL, Supple MA, Surridge A, Tenger-Trolander A, Vogel H, Wilkinson PA, Wilson D, Yorke JA, Yuan F, Balmuth AL, Eland C, Gharbi K, Thomson M, Gibbs RA, Han Y, Jayaseelan JC, Kovar C, Mathew T, Muzny DM, Ongeri F, Pu LL, Qu J, Thornton RL, Worley KC, Wu YQ, Linares M, Blaxter ML, Ffrench-Constant RH, Joron M, Kronforst MR, Mullen SP, Reed RD, Scherer SE, Richards S, Mallet J, Owen McMillan W, Jiggins CD (2012) Butterfly genome reveals promiscuous exchange of mimicry adaptations among species. Nature  
  
DOI[10.1038/nature11041](https://doi.org/10.1038/nature11041)  
EID[2-s2.0-84863496231](https://www.scopus.com/inward/record.uri?eid=2-s2.0-84863496231&doi=10.1038/nature11041&origin=inward)  
PMID[22722851](http://www.ncbi.nlm.nih.gov/pubmed/22722851)  
  
[DOWNLOAD PDF](https://www.igbb.msstate.edu/downloads/Dasmahapatra_et_al_2012.pdf)  
  
ABSTRACT  
The evolutionary importance of hybridization and introgression has long been debated. Hybrids are usually rare and unfit, but even infrequent hybridization can aid adaptation by transferring beneficial traits between species. Here we use genomic tools to investigate introgression in Heliconius, a rapidly radiating genus of neotropical butterflies widely used in studies of ecology, behaviour, mimicry and speciation. We sequenced the genome of Heliconius melpomene and compared it with other taxa to investigate chromosomal evolution in Lepidoptera and gene flow among multiple Heliconius species and races. Among 12,669 predicted genes, biologically important expansions of families of chemosensory and Hox genes are particularly noteworthy. Chromosomal organization has remained broadly conserved since the Cretaceous period, when butterflies split from the Bombyx (silkmoth) lineage. Using genomic resequencing, we show hybrid exchange of genes between three co-mimics, Heliconius melpomene, Heliconius timareta and Heliconius elevatus, especially at two genomic regions that control mimicry pattern. We infer that closely related Heliconius species exchange protective colour-pattern genes promiscuously, implying that hybridization has an important role in adaptive radiation.  

  
  

The Gene Ontology project in 2008  
  

IGBB Authors:  
Shane C. Burgess, Fiona M. McCarthy
PUBLICATION YEAR: 2008  
IMPACT FACTOR: 7.408  
CITATION COUNT: **[688](http://www.scopus.com/inward/citedby.url?partnerID=40&md5=8ab0f0680cf3197f71796d320074d996&origin=inward&eid=2-s2.0-84984996500)**  

more
  
  
  

  

Harris MA, Deegan JI, Ireland A, Lomax J, Ashburner M, Tweedie S, Carbon S, Lewis S, Mungall C, Day-Richter J, Eilbeck K, Blake JA, Bult C, Diehl AD, Dolan M, Drabkin H, Eppig JT, Hill DP, Ni L, Ringwald M, Balakrishnan R, Binkley G, Cherry JM, Christie KR, Costanzo MC, Dong Q, Engel SR, Fisk DG, Hirschman JE, Hitz BC, Hong EL, Krieger CJ, Miyasato SR, Nash, RS, Park, J, Skrzypek, MS, Weng, S, Wong ED, Zhu KK, Botstein D, Dolinski K, Livstone MS, Oughtred R, Berardini T, Li D, Rhee SY, Apweiler R, Barrell D, Camon E, Dimmer E, Huntley R, Mulder N, Khodiyar VK, Lovering RC, Povey S, Chisholm R, Fey P, Gaudet P, Kibbe W, Kishore R, Schwarz EM, Sternberg P, Van Auken K, Giglio MG, Hannick L, Wortman J, Aslett M, Berriman M, Wood V, Jacob H, Laulederkind S, Petri V, Shimoyama M, Smith J, Twigger S, Jaiswal P, Seigfried T, Howe D, Westerfield M, Collmer C, Torto-Alalibo T, Feltrin E, Valle G, Bromberg S, Burgess S, McCarthy F (2008) The Gene Ontology project in 2008. Nucleic Acids Research  
  
DOI[10.1093/nar/gkm883](https://doi.org/10.1093/nar/gkm883)  
EID[2-s2.0-84984996500](https://www.scopus.com/inward/record.uri?eid=2-s2.0-84984996500&doi=10.1093/nar/gkm883&origin=inward)  
PMID[17984083](http://www.ncbi.nlm.nih.gov/pubmed/17984083)  
  
[DOWNLOAD PDF](https://www.igbb.msstate.edu/downloads/Harris_et_al_2008.pdf)  
  
ABSTRACT  
The Gene Ontology (GO) project (http://www.geneontology.org/) provides a set of structured, controlled vocabularies for community use in annotating genes, gene products and sequences (also see http://www.sequenceontology.org/). The ontologies have been extended and refined for several biological areas, and improvements to the structure of the ontologies have been implemented. To improve the quantity and quality of gene product annotations available from its public repository, the GO Consortium has launched a focused effort to provide comprehensive and detailed annotation of orthologous genes across a number of 'reference' genomes, including human and several key model organisms. Software developments include two releases of the ontology-editing tool OBO-Edit, and improvements to the AmiGO browser interface.  

  
  

Comparative genomics reveals insights into avian genome evolution and adaptation  
  

IGBB Authors:  
Federico G. Hoffmann, David A. Ray
PUBLICATION YEAR: 2014  
IMPACT FACTOR: 18.914  
CITATION COUNT: **[780](http://www.scopus.com/inward/citedby.url?partnerID=40&md5=8ab0f0680cf3197f71796d320074d996&origin=inward&eid=2-s2.0-84917678595)**  

more
  
  
  

  

Zhang G, Li C, Li Q, Li B, Larkin DM, Lee C, Storz JF, Antunes A, Greenwold MJ, Meredith RW, Odeen A, Cui J, Zhou Q, Xu L, Pan H, Wang Z, Jin L, Zhang P, Hu H, Yang W, Hu J, Xiao J, Yang Z, Liu Y, Xie Q, Yu H, Lian J, Wen P, Zhang F, Li H, Zeng Y, Xiong Z, Liu S, Zhou L, Huang Z, An N, Wang J, Zheng Q, Xiong Y, Wang G, Wang B, Wang J, Fan Y, da Fonseca RR, Alfaro-Nunez A, Schubert M, Orlando L, Mourier T, Howard JT, Ganapathy G, Pfenning A, Whitney O, Rivas MV, Hara E, Smith J, Farre M, Narayan J, Slavov G, Romanov MN, Borges R, Machado JP, Khan I, Springer MS, Gatesy J, Hoffmann FG, Opazo JC, Hastad O, Sawyer RH, Kim H, Kim KW, Kim HJ, Cho S, Li N, Huang Y, Bruford MW, Zhan X, Dixon A, Bertelsen MF, Derryberry E, Warren W, Wilson RK, Li S, Ray DA, Green RE, O'Brien SJ, Griffin D, Johnson WE, Haussler D, Ryder OA, Willerslev E, Graves GR, Alstrom P, Fjeldsa J, Mindell DP, Edwards SV, Braun EL, Rahbek C, Burt DW, Houde P, Zhang Y, Yang H, Wang J, Jarvis ED, Gilbert MT, Wang J (2014) Comparative genomics reveals insights into avian genome evolution and adaptation. Science  
  
DOI[10.1126/science.1251385](https://doi.org/10.1126/science.1251385)  
EID[2-s2.0-84917678595](https://www.scopus.com/inward/record.uri?eid=2-s2.0-84917678595&doi=10.1126/science.1251385&origin=inward)  
PMID[25504712](http://www.ncbi.nlm.nih.gov/pubmed/25504712)  
  
[DOWNLOAD PDF](https://www.igbb.msstate.edu/downloads/Zhang_et_al_2014.pdf)  
  
ABSTRACT  
Birds are the most species-rich class of tetrapod vertebrates and have wide relevance across many research fields. We explored bird macroevolution using full genomes from 48 avian species representing all major extant clades. The avian genome is principally characterized by its constrained size, which predominantly arose because of lineage-specific erosion of repetitive elements, large segmental deletions, and gene loss. Avian genomes furthermore show a remarkably high degree of evolutionary stasis at the levels of nucleotide sequence, gene synteny, and chromosomal structure. Despite this pattern of conservation, we detected many non-neutral evolutionary changes in protein-coding genes and noncoding regions. These analyses reveal that pan-avian genomic diversity covaries with adaptations to different lifestyles and convergent evolution of traits.  

  
  

Gene ontology annotations and resources  
  

IGBB Authors:  
Susan M. Bridges, Shane C. Burgess, Teresia J. Buza, Fiona M. McCarthy, Lakshmi Pillai
PUBLICATION YEAR: 2013  
IMPACT FACTOR: 8.212  
CITATION COUNT: **[483](http://www.scopus.com/inward/citedby.url?partnerID=40&md5=8ab0f0680cf3197f71796d320074d996&origin=inward&eid=2-s2.0-84874253819)**  

more
  
  
  

  

Blake JA, Dolan M, Drabkin H, Hill DP, Ni L, Sitnikov D, Bridges S, Burgess S, Buza T, McCarthy F, Peddinti D, Pillai L, Carbon S, Dietze H, Ireland A, Lewis SE, Mungall CJ, Gaudet P, Chisholm RL, Fey P, Kibbe WA, Basu S, Siegele DA, McIntosh BK, Renfro DP, Zweifel AE, Hu JC, Brown NH, Tweedie S, Alam-Faruque Y, Apweiler R, Auchinchloss A, Axelsen K, Bely B, Blatter MC, Bonilla C, Bougueleret L, Boutet E, Breuza L, Bridge A, Chan WM, Chavali G, Coudert E, Dimmer E, Estreicher A, Famiglietti L, Feuermann M, Gos A, Gruaz-Gumowski N, Hieta R, Hinz U, Hulo C, Huntley R, James J, Jungo F, Keller G, Laiho K, Legge D, Lemercier P, Lieberherr D, Magrane M, Martin MJ, Masson P, Mutowo-Muellenet P, O'Donovan C, Pedruzzi I, Pichler K, Poggioli D, Porras Millan P, Poux S, Rivoire C, Roechert B, Sawford T, Schneider M, Stutz A, Sundaram S, Tognolli M, Xenarios I, Foulger R, Lomax J, Roncaglia P, Khodiyar VK, Lovering RC, Talmud PJ, Chibucos M, Gwinn Giglio M, Chang HY, Hunter S, McAnulla C, Mitchell A, Sangrador A, Stephan R, Harris MA, Oliver SG, Rutherford K, Wood V, Bahler J, Lock A, Kersey PJ, McDowall MD, Staines DM, Dwinell M, Shimoyama M, Laulederkind S, Hayman T, Wang SJ, Petri V, Lowry T, D'Eustachio P, Matthews L, Balakrishnan R, Binkley G, Cherry JM, Costanzo MC, Dwight SS, Engel SR, Fisk DG, Hitz BC, Hong EL, Karra K, Miyasato SR, Nash RS, Park J, Skrzypek MS, Weng S, Wong ED, Berardini TZ, Li D, Huala E, Mi H, Thomas PD, Chan J, Kishore R, Sternberg P, Van Auken K, Howe D, Westerfield M (2013) Gene ontology annotations and resources. Nucleic Acids Research  
  
DOI[10.1093/nar/gks1050](https://doi.org/10.1093/nar/gks1050)  
EID[2-s2.0-84874253819](https://www.scopus.com/inward/record.uri?eid=2-s2.0-84874253819&doi=10.1093/nar/gks1050&origin=inward)  
PMID[23161678](http://www.ncbi.nlm.nih.gov/pubmed/23161678)  
  
[DOWNLOAD PDF](https://www.igbb.msstate.edu/downloads/Blake_et_al_2013.pdf)  
  
ABSTRACT  
The Gene Ontology (GO) Consortium (GOC, http://www.geneontology.org) is a community-based bio-informatics resource that classifies gene product function through the use of structured, controlled vocabularies. Over the past year, the GOC has implemented several processes to increase the quantity, quality and specificity of GO annotations. First, the number of manual, literature-based annotations has grown at an increasing rate. Second, as a result of a new 'phylogenetic annotation' process, manually reviewed, homology-based annotations are becoming available for a broad range of species. Third, the quality of GO annotations has been improved through a streamlined process for, and automated quality checks of, GO annotations deposited by different annotation groups. Fourth, the consistency and correctness of the ontology itself has increased by using automated reasoning tools. Finally, the GO has been expanded not only to cover new areas of biology through focused interaction with experts, but also to capture greater specificity in all areas of the ontology using tools for adding new combinatorial terms. The GOC works closely with other ontology developers to support integrated use of terminologies. The GOC supports its user community through the use of e-mail lists, social media and web-based resources. -¬ The Author(s) 2012.  

  
  

The genome of the green anole lizard and a comparative analysis with birds and mammals  
  

IGBB Authors:  
David A. Ray, Jeremy D. Smith
PUBLICATION YEAR: 2011  
IMPACT FACTOR: 22.789  
CITATION COUNT: **[531](http://www.scopus.com/inward/citedby.url?partnerID=40&md5=8ab0f0680cf3197f71796d320074d996&origin=inward&eid=2-s2.0-80053359585)**  

more
  
  
  

  

Alfoldi J, Di Palma F, Grabherr M, Williams C, Kong L, Mauceli E, Russell P, Lowe CB, Glor RE, Jaffe JD, Ray DA, Boissinot S, Shedlock AM, Botka C, Castoe TA, Colbourne JK, Fujita MK, Moreno RG, ten Hallers BF, Haussler D, Heger A, Heiman D, Janes DE, Johnson J, de Jong PJ, Koriabine MY, Lara M, Novick PA, Organ CL, Peach SE, Poe S, Pollock DD, de Queiroz K, Sanger T, Searle S, Smith JD, Smith Z, Swofford R, Turner-Maier J, Wade J, Young S, Zadissa A, Edwards SV, Glenn TC, Schneider CJ, Losos JB, Lander ES, Breen M, Ponting CP, Lindblad-Toh K (2011) The genome of the green anole lizard and a comparative analysis with birds and mammals. Nature  
  
DOI[10.1038/nature10390](https://doi.org/10.1038/nature10390)  
EID[2-s2.0-80053359585](https://www.scopus.com/inward/record.uri?eid=2-s2.0-80053359585&doi=10.1038/nature10390&origin=inward)  
PMID[21881562](http://www.ncbi.nlm.nih.gov/pubmed/21881562)  
  
[DOWNLOAD PDF](https://www.igbb.msstate.edu/downloads/Alfoldi_et_al_2011.pdf)  
  
ABSTRACT  
The evolution of the amniotic egg was one of the great evolutionary innovations in the history of life, freeing vertebrates from an obligatory connection to water and thus permitting the conquest of terrestrial environments. Among amniotes, genome sequences are available for mammals and birds, but not for non-avian reptiles. Here we report the genome sequence of the North American green anole lizard, Anolis carolinensis. We find that A. carolinensis microchromosomes are highly syntenic with chicken microchromosomes, yet do not exhibit the high GC and low repeat content that are characteristic of avian microchromosomes. Also, A. carolinensis mobile elements are very young and diverse-more so than in any other sequenced amniote genome. The GC content of this lizard genome is also unusual in its homogeneity, unlike the regionally variable GC content found in mammals and birds. We describe and assign sequence to the previously unknown A. carolinensis X chromosome. Comparative gene analysis shows that amniote egg proteins have evolved significantly more rapidly than other proteins. An anole phylogeny resolves basal branches to illuminate the history of their repeated adaptive radiations.  

  
  

Adverse outcome pathway (AOP) development I: Strategies and principles  
  

IGBB Authors:  
Natàlia Garcia-Reyero
PUBLICATION YEAR: 2014  
IMPACT FACTOR: 5.023  
CITATION COUNT: **[597](http://www.scopus.com/inward/citedby.url?partnerID=40&md5=8ab0f0680cf3197f71796d320074d996&origin=inward&eid=2-s2.0-84921451260)**  

more
  
  
  

  

Villeneuve DL, Crump D, Garcia-Reyero N, Hecker M, Hutchinson TH, LaLone CA, Landesmann B, Lettieri T, Munn S, Nepelska M, Ottinger MA, Vergauwen L, Whelan M (2014) Adverse outcome pathway (AOP) development I: Strategies and principles. Toxicological Sciences  
  
DOI[10.1093/toxsci/kfu199](https://doi.org/10.1093/toxsci/kfu199)  
EID[2-s2.0-84921451260](https://www.scopus.com/inward/record.uri?eid=2-s2.0-84921451260&doi=10.1093/toxsci/kfu199&origin=inward)  
PMID[25466378](http://www.ncbi.nlm.nih.gov/pubmed/25466378)  
  
[DOWNLOAD PDF](https://www.igbb.msstate.edu/downloads/Villeneuve_et_al_2014.pdf)  
  
ABSTRACT  
An adverse outcome pathway (AOP) is a conceptual framework that organizes existing knowledge concerning biologically plausible, and empirically supported, links between molecular-level perturbation of a biological system and an adverse outcome at a level of biological organization of regulatory relevance. Systematic organization of information into AOP frameworks has potential to improve regulatory decision-making through greater integration and more meaningful use of mechanistic data. However, for the scientific community to collectively develop a useful AOP knowledgebase that encompasses toxicological contexts of concern to human health and ecological risk assessment, it is critical that AOPs be developed in accordance with a consistent set of core principles. Based on the experiences and scientific discourse among a group of AOP practitioners, we propose a set of five fundamental principles that guide AOP development: (1) AOPs are not chemical specific; (2) AOPs are modular and composed of reusable components-notably key events (KEs) and key event relationships (KERs); (3) an individual AOP, composed of a single sequence of KEs and KERs, is a pragmatic unit of AOP development and evaluation; (4) networks composed of multiple AOPs that share common KEs and KERs are likely to be the functional unit of prediction for most real-world scenarios; and (5) AOPs are living documents that will evolve over time as new knowledge is generated. The goal of the present article was to introduce some strategies for AOP development and detail the rationale behind these 5 key principles. Consideration of these principles addresses many of the current uncertainties regarding the AOP framework and its application and is intended to foster greater consistency in AOP development.  

  
  
Page 1 of 9 [](https://www.igbb.msstate.edu/citations.php?page=2) [](https://www.igbb.msstate.edu/citations.php?page=9)   
  
  

  
  
  

[More Ways We Can Help You](https://www.igbb.msstate.edu/helpingyou.php)
  

Diane Godwin  
Publications Manager  
GRAPHIC DESIGN  
email  
(662) 325-9333  
Portera  
  

  

  
Topic  
Catfish response to heat stress  
  
IGBB Scientists  
Peter Allen  
Attila Karsi  
  
Funding  
Various sources  

[More IGBB Research](https://www.igbb.msstate.edu/projects.php)
[](https://pa4656.wixsite.com/website)
Dr. PratimaAcharya Adhikari  
Assistant Professor  
[Poultry Science](http://www.poultry.msstate.edu/)  
IGBB Affiliate  
email  
(662) 325-3416  
[website](https://pa4656.wixsite.com/website)
Dr. Peter J.Allen  
Professor  
[Wildlife, Fisheries & Aquaculture](http://www.cfr.msstate.edu/wildlife/index.asp)  
IGBB Affiliate  
email  
(662) 325-4768  

[More Fellows & Affiliates](https://www.igbb.msstate.edu/fellows_affiliates.php)
  

Dr. Mohit Verma  
  
Postdoctoral Associate  
BIOCOMPUTING  
email  
  
Pace 101  
  

#### The IGBB is supported, in part, by the following units:
[   
](http://www.mafes.msstate.edu/)
[   
](http://www.cvm.msstate.edu/)
[   
](http://www.research.msstate.edu/)
[   
](http://www.dafvm.msstate.edu/)
  
The IGBB is an HPC² member center.
