[Mellon College of Science](http://www.cmu.edu/mcs/) › [Biological Sciences](https://www.cmu.edu/bio/index.html) › [ People ](https://www.cmu.edu/bio/people/index.html) › [ Faculty ](https://www.cmu.edu/bio/people/faculty/index.html) › Jonathan S. Minden 
# Jonathan S. Minden
## Professor
  
Address: 279 Mellon InstituteDepartment of Biological SciencesCarnegie Mellon University4400 Fifth AvenuePittsburgh, PA 15213
Phone: 412-268-2669Fax: 412-268-7129
Email
[Lab website](https://labs.bio.cmu.edu/minden/)
#  Education 
Ph.D., Albert Einstein College of Medicine  
Postdoctoral Appointment, University of California, San Francisco
#  Research 
Our lab is interested in how cells and organisms compensate for changes in their environment and in gene expression as a result of mutation. A major driving force in evolution is the organism's ability to adapt to changing environments and genomes. We approach this problem in two ways: (1) by studying how fruit fly (Drosophila) embryos adapt to changes in cell density and fate specification and (2) by using proteomics to study how a variety of organisms adapt to mutational changes.
(1) Pattern Repair: Multicellular organisms develop by transforming a mass of undifferentiated cells into intricately organized groups of cells that coalesce to form functional structures. This process is known as pattern formation. We are interested in what happens when pattern formation is altered: How does the developing embryo respond when too few or too many cells are segregated to form a particular structure? Drosophila embryos have a tremendous capacity to repair mispatterned tissues. Using genetically altered embryos, we can study the cell biology of pattern repair in a wide variety of ways. We have taken an interdisciplinary approach to understanding pattern repair, from genetics and molecular biology to high-end microscopy and computerized image analysis to the design and synthesis of new chemical reagents. We have developed methods to follow cell division, cell shape, gene expression and cell death by time-lapse microscopy. We have also developed a method for turning selected genes on in specific cells with light, called photoactivated gene expression. This allows us to alter the behavior of single cells,or patches of cells, and determine how the cells respond. We have discovered that cell death plays a major role in repairing these induced pattern defects.We are currently investigating how this pattern-repair-induced cell death is controlled.
(2) Proteomics: Proteomics is the analysis of all proteins in a cell, tissue, or organism. We have developed new methods for proteome analysis. Cells constantly have to adapt to environmental changes and mutational changes. This adaptation is carried out by changes in protein expression and protein modification. We have developed a powerful new method to rapidly identify protein differences between cells. This method, called Difference Gel Electrophoresis, involves fluorescently tagging the proteins of different cell extracts with different color fluorescent dyes. The differently colored proteins are mixed and run on the same two-dimensional electrophoresis gel. We then take two pictures of the gel, selecting the wavelength specific for each color. Comparing the images allows one to quickly identify which proteins differ between the samples. We have used this method to identify protein changes during Drosophila embryo development, in Drosophila behavioral mutants, in cancer cells, in mutant yeast and a wide range of organisms and conditions.
#  Publications 
Biedka S, Schmidt BF, Frey NM, Boothman SM, Minden JS, Lucas A. Reversible Click Chemistry Tag for Universal Proteome Sample Preparation for Top-Down and Bottom-Up Analysis. J Proteome Res. 2021 Oct 1;20(10):4787-4800. doi: 10.1021/acs.jproteome.1c00443. Epub 2021 Sep 15. PMID: [34524823](https://pubmed.ncbi.nlm.nih.gov/34524823/)  
  
Jacqueline C, Lee A, Frey N, Minden JS, Finn OJ. Inflammation-Induced Abnormal Expression of Self-molecules on Epithelial Cells: Targets for Tumor Immunoprevention. Cancer Immunol Res. 2020 Aug;8(8):1027-1038. doi: 10.1158/2326-6066.CIR-19-0870. Epub 2020 May 28. PMID: [32467324](https://pubmed.ncbi.nlm.nih.gov/32467324/)  
  
Yablonska S, Ganesan V, Ferrando LM, Kim J, Pyzel A, Baranova OV, Khattar NK, Larkin TM, Baranov SV, Chen N, Strohlein CE, Stevens DA, Wang X, Chang YF, Schurdak ME, Carlisle DL, Minden JS, Friedlander RM. Mutant huntingtin disrupts mitochondrial proteostasis by interacting with TIM23. Proc Natl Acad Sci U S A. 2019 Aug 13;116(33):16593-16602. doi: 10.1073/pnas.1904101116. Epub 2019 Jul 25. PMID: [31346086](https://pubmed.ncbi.nlm.nih.gov/31346086/)  
  
Shorr AZ, Sönmez UM, Minden JS, LeDuc PR. High-throughput mechanotransduction in Drosophila embryos with mesofluidics. Lab Chip. 2019 Mar 27;19(7):1141-1152. doi: 10.1039/c8lc01055b. PMID: [30778467](https://pubmed.ncbi.nlm.nih.gov/30778467/)  
  
Kaupbayeva B, Murata H, Lucas A, Matyjaszewski K, Minden JS, Russell AJ. Molecular Sieving on the Surface of a Nano-Armored Protein. Biomacromolecules. 2019 Mar 11;20(3):1235-1245. doi: 10.1021/acs.biomac.8b01651. Epub 2019 Feb 4. PMID: [30650307](https://pubmed.ncbi.nlm.nih.gov/30650307/)  
  
Blundon M, Ganesan V, Redler B, Van PT, Minden JS. Two-Dimensional Difference Gel Electrophoresis. Methods Mol Biol. 2019;1855:229-247. doi: 10.1007/978-1-4939-8793-1_20. PMID: [30426421](https://pubmed.ncbi.nlm.nih.gov/30426421/)  
  
Blundon MA, Schlesinger DR, Parthasarathy A, Smith SL, Kolev HM, Vinson DA, Kunttas-Tatli E, McCartney BM, Minden JS. Proteomic analysis reveals APC-dependent post-translational modifications and identifies a novel regulator of β-catenin. Development. 2016 Jul 15;143(14):2629-40. doi: 10.1242/dev.130567. Epub 2016 Jun 10. PMID: [27287809](https://pubmed.ncbi.nlm.nih.gov/27287809/)
Ganesan V, Ascherman DP, Minden JS. Immunoproteomics technologies in the discovery of autoantigens in autoimmune diseases. [Biomol Concepts. 2016 May 1;7(2):133-43.](http://www.ncbi.nlm.nih.gov/pubmed/27115324/)  
  
Ganesan V, Schmidt B, Avula R, Cooke D, Maggiacomo T, Tellin L, Ascherman DP, Bruchez MP, Minden J. Immuno-proteomics: Development of a novel reagent for separating antibodies from their target proteins. [Biochim Biophys Acta. 2015 Jun;1854(6):592-600.](http://www.ncbi.nlm.nih.gov/pubmed/25466873/)  
  
Van PT, Ganesan V, Bass V, Parthasarathy A, Schlesinger D, Minden JS. In-gel equilibration for improved protein retention in 2DE-based proteomic workflows. [Electrophoresis. 2014 Jul 17](http://www.ncbi.nlm.nih.gov/pubmed/25042010)  
  
Van PT, Bass V, Shiwarski D, Lanni F, Minden J. High dynamic range proteome imaging with the Structured Illumination Gel Imager. [Electrophoresis. 2014 Jun 17.](http://www.ncbi.nlm.nih.gov/pubmed/24935033)  
  
Iheagwara UK, Beatty PL, Van PT, Ross TM, Minden JS, Finn OJ. Influenza virus infection elicits protective antibodies and T cells specific for host cell antigens also expressed as tumor-associated antigens: a new view of cancer immunosurveillance. [Cancer Immunol Res. 2014 Mar;2(3):263-73.](http://www.ncbi.nlm.nih.gov/pubmed/24778322)  
  
Krajcovic MM, Minden JS. Assessing the critical period for Rho kinase activity during Drosophila ventral furrow formation. [Dev Dyn. 2012 Sep 12.](http://www.ncbi.nlm.nih.gov/pubmed/22972587)  
[](http://www.ncbi.nlm.nih.gov/pubmed/18931788?ordinalpos=1&itool=EntrezSystem2.PEntrez.Pubmed.Pubmed_ResultsPanel.Pubmed_DefaultReportPanel.Pubmed_RVDocSum)
Minden JS. DIGE: Past and Future. [Methods Mol Biol. 2012;854:3-8.](http://www.ncbi.nlm.nih.gov/pubmed/22311749)  
  
Fishilevich E, Fitzpatrick JAJ, Minden JS. pHMA, a pH-sensitive GFP reporter for cell engulfment in Drosophila[Dev. Dyn. 238, 559-73, 2010](http://www.ncbi.nlm.nih.gov/pubmed/20014104).
Monzo K, Dowd SR, Minden JS, Sisson JS. Proteomic analysis reveals CCT is a target of Fragile X mental retardation protein regulation in Drosophila. [Dev. Biol. 16, 2891-902, 2010.](http://www.ncbi.nlm.nih.gov/pubmed/20122915)
Minden JS, Dowd SR, Meyer HE, Stühler K. Difference Gel Electrophoresis. [Electrophoresis. 30, S156-S161, 2009.](http://www.ncbi.nlm.nih.gov/pubmed/19517495)
Puri M, Goyal A, Senutovich N, Dowd SR, Minden JS. Building proteomic pathways using Drosophila ventral furrow formation as a model. [Mol Biosyst. 1:1126-35. Epub 2008 Sep 26, 2008.](http://www.ncbi.nlm.nih.gov/pubmed/18931788?ordinalpos=1&itool=EntrezSystem2.PEntrez.Pubmed.Pubmed_ResultsPanel.Pubmed_DefaultReportPanel.Pubmed_RVDocSum)
Witzberger MM, Fitzpatrick JA, Crowley JC, Minden JS. End-on imaging: A new perspective on dorsoventral development in Drosophila embryos. [Dev Dyn. 237(11):3252-3259, 2008.](http://www.ncbi.nlm.nih.gov/pubmed/18855896?ordinalpos=2&itool=EntrezSystem2.PEntrez.Pubmed.Pubmed_ResultsPanel.Pubmed_DefaultReportPanel.Pubmed_RVDocSum)
Minden JS. Dissection of the embryonic brain using photoactivated gene expression. [Adv Exp Med Biol. 628:57-68, 2008.](http://www.ncbi.nlm.nih.gov/pubmed/18683638?ordinalpos=2&itool=EntrezSystem2.PEntrez.Pubmed.Pubmed_ResultsPanel.Pubmed_DefaultReportPanel.Pubmed_RVDocSum)
Minden JS. Comparative proteomics and difference gel electrophoresis. [Biotechniques 43(6):739, 741, 743 passim, 2007.](http://www.ncbi.nlm.nih.gov/pubmed/18251249?ordinalpos=5&itool=EntrezSystem2.PEntrez.Pubmed.Pubmed_ResultsPanel.Pubmed_DefaultReportPanel.Pubmed_RVDocSum)
Boyce M, Py BF, Ryazanov AG, Minden JS, Long K, Ma D, Yuan J. A pharmacoproteomic approach implicates eukaryotic elongation factor 2 kinase in ER stress-induced cell death. [Cell Death Differ 15(3):589-99. Epub 2008 Jan 11.](http://www.ncbi.nlm.nih.gov/pubmed/18188169?ordinalpos=6&itool=EntrezSystem2.PEntrez.Pubmed.Pubmed_ResultsPanel.Pubmed_DefaultReportPanel.Pubmed_RVDocSum)
Sellers KF, Miecznikowski J, Viswanathan S, Minden JS, Eddy WF. Lights, Camera, Action! -- Systematic Variation in 2-D Difference Gel Electrophoresis Images. [Electrophoresis 18:3324-32, 2007.](http://www.ncbi.nlm.nih.gov/pubmed/17854127?ordinalpos=4&itool=EntrezSystem2.PEntrez.Pubmed.Pubmed_ResultsPanel.Pubmed_DefaultReportPanel.Pubmed_RVDocSum)
Minden JS. Dissection of the embryonic brain using photoactivated gene expression. In Brain Development in Drosophila
Viswanathan S, Ünlü M, Minden JS. Two-dimensional difference gel electrophoresis. [Nat. Protocols 1:1351-1358, 2006.](http://www.ncbi.nlm.nih.gov/pubmed/17406422?ordinalpos=8&itool=EntrezSystem2.PEntrez.Pubmed.Pubmed_ResultsPanel.Pubmed_DefaultReportPanel.Pubmed_RVDocSum)
Nakahara K, Kim K, Sciulli C, Dowd SR, Minden JS, Carthew RW. Targets of microRNA regulation in the Drosophila oocyte proteome. [Proc Natl Acad Sci U S A. 102:12023-12028, 2005.](http://www.ncbi.nlm.nih.gov/pubmed/16099838?ordinalpos=9&itool=EntrezSystem2.PEntrez.Pubmed.Pubmed_ResultsPanel.Pubmed_DefaultReportPanel.Pubmed_RVDocSum)
Gong L, Puri M, Ünlü M, Young M, Robertson K, Viswanathan S, Krishnaswamy A, Dowd SR, Minden JS. Drosophila Ventral Furrow Morphogenesis: A Proteomic Analysis. [Development. 131:643-656, 2004.](http://www.ncbi.nlm.nih.gov/pubmed/14711877?ordinalpos=10&itool=EntrezSystem2.PEntrez.Pubmed.Pubmed_ResultsPanel.Pubmed_DefaultReportPanel.Pubmed_RVDocSum)
Mergliano J, Minden JS. Caspase-independent cell engulfment mirrors cell death pattern in Drosophila embryos. [Development. 130:5779-5789, 2003.](http://www.ncbi.nlm.nih.gov/pubmed/14534140?ordinalpos=11&itool=EntrezSystem2.PEntrez.Pubmed.Pubmed_ResultsPanel.Pubmed_DefaultReportPanel.Pubmed_RVDocSum)
Robertson K, Mergliano J, Minden JS. Dissecting Drosophila embryonic brain development using photoactivated gene expression. [Developmental Biology 260:124-137, 2003.  
](http://www.ncbi.nlm.nih.gov/pubmed/12885560?ordinalpos=12&itool=EntrezSystem2.PEntrez.Pubmed.Pubmed_ResultsPanel.Pubmed_DefaultReportPanel.Pubmed_RVDocSum)
[Full PubMed Listings](http://www.ncbi.nlm.nih.gov/pubmed/?term=Minden+JS)
  * [News](https://www.cmu.edu/bio/news/index.html)
  * [Alumni](https://www.cmu.edu/bio/people/alumni/index.html)
  * [Mellon College of Science](http://www.cmu.edu/mcs/)


  * [Directory](https://www.cmu.edu/bio/people/index.html)
  * [Contact](https://www.cmu.edu/bio/visit/index.html)
  * [Courses](https://www.cmu.edu/bio/courses/index.html)


  * [Faculty](https://www.cmu.edu/bio/people/faculty/index.html)
  * [Administration](https://www.cmu.edu/bio/people/admin.html)


  * [Forms](https://www.cmu.edu/bio/graduate/current_students/forms/index.html)
  * [Intranet](https://www.bio.cmu.edu/)


