[Mellon College of Science](http://www.cmu.edu/mcs/) › [Biological Sciences](https://www.cmu.edu/bio/index.html) › [ People ](https://www.cmu.edu/bio/people/index.html) › [ Faculty ](https://www.cmu.edu/bio/people/faculty/index.html) › Dannie Durand 
# Dannie Durand
## Associate Professor
  
Address: 646 Mellon InstituteDepartment of Biological SciencesCarnegie Mellon University4400 Fifth AvenuePittsburgh, PA 15213
Phone: 412-268-6036Fax: 412-268-7129
Email  
[Lab website](http://www-2.cs.cmu.edu/~durand/Lab/)
#  Education 
Ph.D. Computer Science, Columbia University
### Postdoctoral Appointments
University of Pennsylvania Computational Biology Group  
Sloan Computational Biology Fellowship, Molecular Biology, Princeton University
#  Research 
Computational analysis of molecular sequence data is a key component in solving three critical biological problems of the 21st century: how genes interact to produce living cells, how gene malfunction causes disease and how complex, multicellular organisms evolved from simple, unicellular organisms. In my research, I use computational approaches to studying the role of gene duplication in the acquisition of new gene function and the evolution of vertebrate genomes. New genes arise through gene duplications, errors during cell division that result in extra copies of genes. These extra copies subsequently mutate to take on new functional roles in the cell. The duplication of large regions, ranging from chromosomal segments to the entire genome, is believed to have played a crucial role in early vertebrate evolution. According to the hypothesis, the new genes that resulted from these massive duplications are responsible for the evolution of innovations, such as skeletal structure, limbs, and a complex central nervous system, that distinguish vertebrates from other life forms. If we can understand how these genes acquired new function following duplication, we will have a better understanding of how we evolved and the role those genes play in vertebrates living today.
#  Publications 
Cui X, Xue Y, McCormack C, Garces A, Rachman TW, Yi Y, Stolzer M, Durand D. Simulating domain architecture evolution. Bioinformatics. 2022 Jun 24;38(Suppl 1):i134-i142. doi: 10.1093/bioinformatics/btac242. PMID: [35758772](https://pubmed.ncbi.nlm.nih.gov/35758772/); PMCID: PMC9236583
Davidson P, Eutsey R, Redler B, Hiller NL, Laub MT, Durand D. Flexibility and constraint: Evolutionary remodeling of the sporulation initiation pathway in Firmicutes. PLoS Genet. 2018 Sep 13;14(9):e1007470. doi: 10.1371/journal.pgen.1007470. PMID: [30212463](https://pubmed.ncbi.nlm.nih.gov/30212463/)  
  
Xu M, Lawrence JG, Durand D. Selection, periodicity and potential function for Highly Iterative Palindrome-1 (HIP1) in cyanobacterial genomes. Nucleic Acids Res. 2018 Mar 16;46(5):2265-2278. doi: 10.1093/nar/gky075. PMID: [29432573](https://pubmed.ncbi.nlm.nih.gov/29432573/)  
  
Antic I, Brothers KM, Stolzer M, Lai H, Powell E, Eutsey R, Cuevas RA, Miao X, Kowalski RP, Shanks RMQ, Durand D, Hiller NL. Gene Acquisition by a Distinct Phyletic Group within Streptococcus pneumoniae Promotes Adhesion to the Ocular Epithelium. mSphere. 2017 Oct 25;2(5):e00213-17. doi: 10.1128/mSphere.00213-17. PMID: [29085912](https://pubmed.ncbi.nlm.nih.gov/29085912/)
Darby, C.A., Stolzer, M., Ropp, P. J., Barker, D. & Durand, D. (2017). Xenolog classification. [Bioinformatics, 33 (5): 640-649. doi:10.1093/bioinformatics/btw686.](https://academic.oup.com/bioinformatics/article/33/5/640/2725487/Xenolog-classification)
M. Stolzer, K. Siewert, H. Lai, M. Xu, D. Durand, 2015. Event inference in multidomain families with phylogenetic reconciliation. [BMC Bioinformatics, Oct 2;16 Suppl 14:S8](http://www.ncbi.nlm.nih.gov/pubmed/26451642)  
  
Stolzer M, Wasserman L, Durand D. Robustness of birth-death and gain models for inferring evolutionary events. [BMC Genomics 2014, 15(Suppl 6):S9](http://www.biomedcentral.com/1471-2164/15/S6/S9)  
  
Capra JA, Stolzer M§, Durand D*, Pollard KS. How old is my gene? [Trends in Genetics. Volume 29, Issue 11.](http://www.sciencedirect.com/science/article/pii/S016895251300111X)  
  
Stolzer M, Lai H, Xu M, Sathaye D, Vernot B, Durand D. Inferring duplications, losses, transfers and incomplete lineage sorting with nonbinary species trees. [Bioinformatics. 2012 Sep 15;28(18):i409-i415](http://www.ncbi.nlm.nih.gov/pubmed/22962460).
Livstone M, Oughtred R, Heinicke S, Vernot B, Huttenhower C, Durand D, Dolinski K. Inferring protein function from homology using the Princeton Protein Orthology Database (P-POD). Curr Protoc Bioinformatics, Chapter 6: Unit 6:11, 2011.
Raghupathy N and Durand D. Gene cluster statistics with gene families. [Molecular Biology and Evolution, 26:957-968, 2009](http://www.ncbi.nlm.nih.gov/pubmed/19150803).
Joseph JM and Durand D. Family classification without domain chaining. [Bioinformatics 25(12):i45-i53, 2009](http://bioinformatics.oxfordjournals.org/cgi/content/short/25/12/i45).
Vernot B, Stolzer M, Goldman A and Durand D. Reconciliation with non-binary species trees. [Journal of Computational Biology, 15(8): 981-1006, 2008](http://www.ncbi.nlm.nih.gov/pubmed/18808330?ordinalpos=1&itool=EntrezSystem2.PEntrez.Pubmed.Pubmed_ResultsPanel.Pubmed_DefaultReportPanel.Pubmed_RVDocSum).
Song N, Joseph JM, Davis GB and Durand D. Sequence similarity network reveals common ancestry of multidomain proteins. [PLOS Computational Biology, 4(5): e1000063, 2008](http://www.ncbi.nlm.nih.gov/pubmed/18475320?ordinalpos=1&itool=EntrezSystem2.PEntrez.Pubmed.Pubmed_ResultsPanel.Pubmed_DefaultReportPanel.Pubmed_RVDocSum).
Raghupathy N, Hoberman R and Durand D. Two plus two does not equal three: Statistical tests for multiple genome comparison. [Journal of Bioinformatics and Computational Biology, 6(1): 1-22, 2008](http://www.ncbi.nlm.nih.gov/pubmed/18324742?ordinalpos=12&itool=EntrezSystem2.PEntrez.Pubmed.Pubmed_ResultsPanel.Pubmed_DefaultReportPanel.Pubmed_RVDocSum).
Song N, Sedgewick RD and Durand D. Domain architecture comparison for multidomain homology identification. [Journal of Computational Biology, 14(4): 496-516, 2007](http://www.ncbi.nlm.nih.gov/pubmed/17572026?ordinalpos=2&itool=EntrezSystem2.PEntrez.Pubmed.Pubmed_ResultsPanel.Pubmed_DefaultReportPanel.Pubmed_RVDocSum).
Durand D and Hoberman R. Diagnosing duplications: can it be done? [Trends in Genetics, 22(3):156-64, 2006 Mar](http://www.ncbi.nlm.nih.gov/pubmed/16442663).
Durand D, Halldorsson BV and Vernot B. Hybrid micro-macroevolutionary approach to gene tree reconstruction. [Journal of Computational Biology, 13 (2): 320-335, 2006](http://www.ncbi.nlm.nih.gov/pubmed/16597243?ordinalpos=8&itool=EntrezSystem2.PEntrez.Pubmed.Pubmed_ResultsPanel.Pubmed_DefaultReportPanel.Pubmed_RVDocSum).
Przytycka T, Davis GB, Song N and Durand D. Graph theoretical insights into evolution of multidomain proteins. [Journal of Computational Biology, 13 (2): 351-363, 2006](http://www.ncbi.nlm.nih.gov/pubmed/16597245?dopt=Abstract).
Hoberman R, Sankoff D and Durand D. The statistical analysis of spatially clustered genes under the maximum gap criterion. [Journal of Computational Biology, 12(8): 1081-1100, 2005](http://www.ncbi.nlm.nih.gov/pubmed/16241899?ordinalpos=119&itool=EntrezSystem2.PEntrez.Pubmed.Pubmed_ResultsPanel.Pubmed_DefaultReportPanel.Pubmed_RVDocSum).
Durand D. Vertebrate evolution: doubling and shuffling with a full deck. [Trends in Genetics, 19(1):2-5, 2003 Jan](http://www.ncbi.nlm.nih.gov/pubmed/12493240).
Durand D and Sankoff D. Tests for gene clustering. [Journal of Computational Biology, 10 (3/4): 453–482, 2003](http://www.ncbi.nlm.nih.gov/pubmed/12935338?dopt=Abstract).
Chen K, Durand D and Farach-Colton M. Notung: A program for dating gene duplications and optimizing gene family trees. [Journal of Computational Biology, 7 (3/4):429–447, 2000](http://www.ncbi.nlm.nih.gov/pubmed/11108472?ordinalpos=2&itool=EntrezSystem2.PEntrez.Pubmed.Pubmed_ResultsPanel.Pubmed_DefaultReportPanel.Pubmed_RVDocSum).
  * [News](https://www.cmu.edu/bio/news/index.html)
  * [Alumni](https://www.cmu.edu/bio/people/alumni/index.html)
  * [Mellon College of Science](http://www.cmu.edu/mcs/)


  * [Directory](https://www.cmu.edu/bio/people/index.html)
  * [Contact](https://www.cmu.edu/bio/visit/index.html)
  * [Courses](https://www.cmu.edu/bio/courses/index.html)


  * [Faculty](https://www.cmu.edu/bio/people/faculty/index.html)
  * [Administration](https://www.cmu.edu/bio/people/admin.html)


  * [Forms](https://www.cmu.edu/bio/graduate/current_students/forms/index.html)
  * [Intranet](https://www.bio.cmu.edu/)


