[Mellon College of Science](http://www.cmu.edu/mcs/) › [Biological Sciences](https://www.cmu.edu/bio/index.html) › [ People ](https://www.cmu.edu/bio/people/index.html) › [ Faculty ](https://www.cmu.edu/bio/people/faculty/index.html) › C. Joel McManus 
# C. Joel McManus
## Associate Professor
  
Address: 255 Mellon InstituteDepartment of Biological SciencesCarnegie Mellon University4400 Fifth AvenuePittsburgh, PA 15213
Phone: 412-268-9407Fax: 412-268-7129
Email
[Lab website](https://labs.bio.cmu.edu/mcmanus/)
#  Education 
Ph.D., University of Wisconsin, Madison  
Postdoctoral Appointment, University of Connecticut Health Center
#  Research 
How does variation in gene expression contribute to phenotypic diversity and disease? Gene expression involves transcription of DNA into mRNA, alternative splicing of mRNA, translation of mRNA into proteins, and regulation of mRNA and protein levels through turnover pathways. Research in the McManus lab focuses on understanding mechanisms that regulate mRNA translation in fungal species, and how variation in RNA sequences and structures affects protein production. Our lab develops and employs novel high-throughput assay systems to identify RNA cis  
  
1. How do uORFs control translation? Upstream Open Reading Frames (uORFs) are short open reading frames found in mRNA transcript leaders that regulate mRNA translation and stability in eukaryotes. These fascinating regulatory elements control the expression of many eukaryotic genes. Intriguingly, many uORFs initiate at non-AUG (“near-cognate”) start codons (e.g. UUG). We use quantitative analysis of ribosome profiling to identify candidate uORFs in the yeast Saccharomyces cerevisiae and its sister species.
This footprinting technique identifies the locations on ribosomes on mRNA genome wide. By computationally assessing the resulting profiles of ribosome occupancy, we have predicted thousands of candidate uORFs in three Saccharomyces
2. How is translation regulated by a fungal pathogen during infection of a mammalian host? Candida albicansC. albicans
#  Publications 
Akirtava C, McManus CJ. Control of translation by eukaryotic mRNA transcript leaders-Insights from high-throughput assays and computational modeling. Wiley Interdiscip Rev RNA. 2021 May;12(3):e1623. doi: 10.1002/wrna.1623. Epub 2020 Aug 31. PMID: 32869519Lagree K, Woolford CA, Huang MY, May G, McManus CJ, Solis NV, Filler SG, Mitchell AP. Roles of Candida albicans Mig1 and Mig2 in glucose repression, pathogenicity traits, and SNF1 essentiality. PLoS Genet. 2020 Jan 21;16(1):e1008582. doi: 10.1371/journal.pgen.1008582. PMID: 31961865Lin Y, May GE, Kready H, Nazzaro L, Mao M, Spealman P, Creeger Y, McManus CJ. Impacts of uORF codon identity and position on translation regulation. Nucleic Acids Res. 2019 Sep 26;47(17):9358-9367. doi: 10.1093/nar/gkz681. PMID: 31392980
Huang MY, Woolford CA, May G, McManus CJ, Mitchell AP. Circuit diversification in a biofilm regulatory network. PLoS Pathog. 2019 May 22;15(5):e1007787. doi: 10.1371/journal.ppat.1007787. eCollection 2019 May. PMID: 31116789Lin Y, Schmidt BF, Bruchez MP, McManus CJ. Structural analyses of NEAT1 lncRNAs suggest long-range RNA interactions that may contribute to paraspeckle architecture. Nucleic Acids Res. 2018 Apr 20;46(7):3742-3752. doi: 10.1093/nar/gky046. PMID: 29394378Wang H, Kingsford C, McManus CJ. Using the Ribodeblur pipeline to recover A-sites from yeast ribosome profiling data. Methods. 2018 Mar 15;137:67-70. doi: 10.1016/j.ymeth.2018.01.002. Epub 2018 Jan 9. PMID: 29330118
Spealman P, Naik AW, May GE, Kuersten S, Freeberg L, Murphy RF, McManus J. Conserved non-AUG uORFs revealed by a novel regression analysis of ribosome profiling data. Genome Res. 2018 Feb;28(2):214-222. doi: 10.1101/gr.221507.117. Epub 2017 Dec 18. PMID: 29254944Wang H, McManus J, Kingsford C. Accurate Recovery of Ribosome Positions Reveals Slow Translation of Wobble-Pairing Codons in Yeast. J Comput Biol. 2017 Jun;24(6):486-500. doi: 10.1089/cmb.2016.0147. Epub 2016 Oct 11. PMID: 27726445
Wang H, McManus J, Kingsford C. Isoform-level ribosome occupancy estimation guided by transcript abundance with Ribomap. Bioinformatics. 2016 Jun 15;32(12):1880-2. doi: 10.1093/bioinformatics/btw085. Epub 2016 Feb 15. PMID: 27153676[](http://www.ncbi.nlm.nih.gov/pubmed/26068740)
McManus J, Cheng Z, Vogel C. Next-generation analysis of gene expression regulation--comparing the roles of synthesis and degradation. Mol Biosyst. 2015 Oct;11(10):2680-9. doi: 10.1039/c5mb00310e. Review. PMID: 26259698
Lin Y, May GE, Joel McManus C. Mod-seq: A High-Throughput Method for Probing RNA Secondary Structure. [Methods Enzymol. 2015;558:125-52.](http://www.ncbi.nlm.nih.gov/pubmed/26068740)
Coolon JD, Stevenson KR, McManus CJ, Yang B, Graveley BR, Wittkopp PJ. Molecular mechanisms and evolutionary processes contributing to accelerated divergence of gene expression on the Drosophila X chromosome. Mol Biol Evol. 2015 Jun 2.
McManus CJ, May GE, Spealman P, Shteyman A. Ribosome profiling reveals post-transcriptional buffering of divergent gene expression in yeast. Genome Res. [2014 Mar;24(3):422-30.](http://www.ncbi.nlm.nih.gov/pubmed/24318730)  
  
Rafiq K, Shashikant T, McManus CJ, Ettensohn CA. Genome-wide analysis of the skeletogenic gene regulatory network of sea urchins. [Development. 2014 Feb;141(4):950-61.](http://www.ncbi.nlm.nih.gov/pubmed/24496631)  
  
Dembowski J.A., Ramesh M, McManus C. J., and Woolford J.L. Jr. (2013) Identification of the binding site of Rlp7 on assembling 60S ribosomal subunits in Saccharomyces cerevisiae. [RNA 19:1639-1647.](http://rnajournal.cshlp.org/content/19/12/1639.long)
McManus CJ, Graveley BR. [Curr Opin Genet Dev 21(4)](http://www.sciencedirect.com/science/article/pii/S0959437X11000694)[:373-9,2011](http://www.sciencedirect.com/science/article/pii/S0959437X11000694).
May GE, Olson S, McManus CJ, and Graveley BR. Competing RNA secondary structures are required for mutually exclusive splicing of the Dscam[RNA 17(2):222-229,2011](http://rnajournal.cshlp.org/content/17/2/222.full?sid=79d47c12-8d10-40b7-96cc-c07495b7f2b4).
McManus CJ, Duff MO, Eipper-Mains J, and Graveley BR. Global analysis of transDrosophila[Proc Natl Acad Sci USA 107(29):12975-12979, 2010](http://www.pnas.org/content/107/29/12975.long).
McManus CJ, Coolon JD, Duff MO, Eipper-Mains J, Graveley BR, and Wittkopp PJ. Drosophila[Genome Res 20(6)](http://genome.cshlp.org/content/20/6/816.long).
McManus CJ, Graveley BR. [Mol Cell. 31(1):](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2713004/?tool=pubmed).
[Full PubMed Listings](http://www.ncbi.nlm.nih.gov/pubmed/?term=Joel+McManus)
  * [News](https://www.cmu.edu/bio/news/index.html)
  * [Alumni](https://www.cmu.edu/bio/people/alumni/index.html)
  * [Mellon College of Science](http://www.cmu.edu/mcs/)


  * [Directory](https://www.cmu.edu/bio/people/index.html)
  * [Contact](https://www.cmu.edu/bio/visit/index.html)
  * [Courses](https://www.cmu.edu/bio/courses/index.html)


  * [Faculty](https://www.cmu.edu/bio/people/faculty/index.html)
  * [Administration](https://www.cmu.edu/bio/people/admin.html)


  * [Forms](https://www.cmu.edu/bio/graduate/current_students/forms/index.html)
  * [Intranet](https://www.bio.cmu.edu/)


